pandapower/toolbox.py

```diff
@@ -2041,7 +2041,7 @@ def replace_zero_branches_with_switches(net, elements=('line', 'impedance'), zer
             net[elm].loc[b, 'in_service'] = False
             affected_elements.add(b)
 
-        replaced[elm] = net[elm].loc[affected_elements]
+        replaced[elm] = net[elm].loc[list(affected_elements)]
```

Comment:
also should be possible with numpy

Reply:
but due to performance aspects, because affected elements is initialized as set and used as set, the code here is faster than numpy operations